modules/UI.js

```diff
@@ -0,0 +1,74 @@
+import Storage from './Storage';
+
+class UI {
+  static addTask(task) {
+    const tasksList = document.querySelector('ul');
+    const taskList = document.createElement('li');
+    taskList.setAttribute('class', 'list-group-item d-flex align-items-start');
+    const taskListContent = `
+                <div class="${(task.completed === false) ? '' : 'text-decoration-line-through'} d-flex ms-2 p-2 me-auto fs-3">
+                  <input
+                    class="form-check-input me-1"
+                    ${(task.completed === false) ? '' : 'checked'}
+                    type="checkbox"
+                    aria-label="..."
+                    autofocus
+                  />
+                  <div class="ps-3 pe-5 task-description">${task.description}</div>
+                </div>
+                <i class="btn text-primary fs-3 bi bi-three-dots-vertical"></i>
+            `;
+    taskList.innerHTML = taskListContent;
+    tasksList.appendChild(taskList);
+  }
+
+  static displayTasks(tasks) {
```

Comment:
- [ ] As written in the project requirements. I kindly suggest you please remove all hardcoded  items from the tasks array
![image](https://user-images.githubusercontent.com/52934128/177671294-3b8adad3-b561-4ea1-ab7b-f4ffbfb71e88.png)